I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/screens.rpy", line 1179, in execute
    screen confirm(message, yes_action, no_action):
  File "game/screens.rpy", line 1179, in execute
    screen confirm(message, yes_action, no_action):
  File "game/screens.rpy", line 1192, in execute
    frame:
  File "game/screens.rpy", line 1194, in execute
    vbox:
  File "game/screens.rpy", line 1203, in execute
    hbox:
  File "game/screens.rpy", line 1209, in execute
    imagebutton:
Exception: Imagebutton does not have a idle image. (auto='YesButton_%s.png').

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in script
    $ ui.interact()
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_load_save.rpym", line 35, in <module>
    $ ui.interact()
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 3579, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 4052, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/screen.py", line 476, in visit_all
    callback(self)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/core.py", line 4052, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/screen.py", line 487, in per_interact
    self.update()
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/display/screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 1179, in execute
    screen confirm(message, yes_action, no_action):
  File "game/screens.rpy", line 1179, in execute
    screen confirm(message, yes_action, no_action):
  File "game/screens.rpy", line 1192, in execute
    frame:
  File "game/screens.rpy", line 1194, in execute
    vbox:
  File "game/screens.rpy", line 1203, in execute
    hbox:
  File "game/screens.rpy", line 1209, in execute
    imagebutton:
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/ui.py", line 948, in _imagebutton
    idle = choice(idle, idle_image, "idle", required=True)
  File "/Users/akpulaski/Applications/renpy-8.1.0-sdk/renpy/ui.py", line 942, in choice
    raise Exception("Imagebutton does not have a %s image. (auto=%r)." % (name, auto))
Exception: Imagebutton does not have a idle image. (auto='YesButton_%s.png').

macOS-13.6-arm64-arm-64bit arm64
Ren'Py 8.1.1.23060707
THRALL 0.0
Sun Oct  1 14:01:19 2023
